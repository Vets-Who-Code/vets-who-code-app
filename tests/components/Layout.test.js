import { render } from '@testing-library/react'
import TemplateWrapper from '@/components/Layout'

describe('<Layout />', () => {
  test('should render correctly', () => {
    const { container } = render(<TemplateWrapper />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
