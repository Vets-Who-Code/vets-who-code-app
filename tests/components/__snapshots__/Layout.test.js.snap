// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> should render correctly 1`] = `
<div>
  <main
    class="main_container"
  >
    <div
      class="Toastify"
    />
    <nav
      class="navbar navbar-fixed-top main-navigation navbar-solid"
      id="fixedTopNav"
      itemtype="https://schema.org/SiteNavigationElement"
      role="navigation"
      style="opacity: 0.9;"
    >
      <div
        class="container-fluid"
      >
        <div
          class="navbar-header"
        >
          <div
            class="navbar-brand"
            itemtype="https://schema.org/Organization"
            style="padding: 5px 15px 5px;"
          >
            <span
              class="sr-only"
            >
              #VetsWhoCode
            </span>
            <a
              href="/"
              id="navbar-brand"
            >
              <img
                alt="#VetsWhoCode Logo"
                class="logo_holder"
                height="40"
                src="test-file-stub"
                width="40"
              />
              <div
                class="homeLink"
              >
                VetsWhoCode
              </div>
            </a>
          </div>
          <button
            aria-expanded="false"
            class="navbar-toggle collapsed hamburger "
            id="hamburger-1"
            type="button"
          >
            <span
              class="line"
            />
            <span
              class="line"
            />
            <span
              class="line"
            />
            <span
              class="sr-only"
            >
              #VetsWhoCode
            </span>
          </button>
        </div>
        <div
          aria-expanded="false"
          class="navbar-collapse collapse "
          id="main-nav-collapse"
        >
          <ul
            class="nav navbar-nav navbar-right"
            id="navbar-list"
            role="menu"
          >
            <li
              class="nav"
              role="menuitem"
            >
              <span />
            </li>
            <li
              class="nav"
              role="menuitem"
            >
              <a
                href="/"
              >
                <span>
                  Home
                </span>
              </a>
            </li>
            <li
              class="dropdown nav "
              role="menuitem"
              style="margin-right: 10px;"
            >
              <a
                aria-expanded="false"
                aria-haspopup="true"
                class="nav"
                data-toggle="dropdown"
                href="#"
                role="button"
              >
                Our Story
                <svg
                  class="dropdown-caret"
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  stroke-width="0"
                  viewBox="0 0 1024 1024"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M840.4 300H183.6c-19.7 0-30.7 20.8-18.5 35l328.4 380.8c9.4 10.9 27.5 10.9 37 0L858.9 335c12.2-14.2 1.2-35-18.5-35z"
                  />
                </svg>
              </a>
              <ul
                class="dropdown-menu"
              >
                <li
                  class="nav media-dropdown-item"
                  role="menuitem"
                >
                  <a
                    href="/about"
                  >
                    <span>
                      About
                    </span>
                  </a>
                </li>
                <li
                  class="nav media-dropdown-item"
                  role="menuitem"
                >
                  <a
                    href="/board"
                  >
                    <span>
                      Board
                    </span>
                  </a>
                </li>
                <li
                  class="nav"
                  role="menuitem"
                >
                  <a
                    href="/testimonials"
                  >
                    <span>
                      Testimonials
                    </span>
                  </a>
                </li>
              </ul>
            </li>
            <li
              class="dropdown nav "
              role="menuitem"
              style="margin-right: 10px;"
            >
              <a
                aria-expanded="false"
                aria-haspopup="true"
                class="nav"
                data-toggle="dropdown"
                href="#"
                role="button"
              >
                Media
                <svg
                  class="dropdown-caret"
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  stroke-width="0"
                  viewBox="0 0 1024 1024"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M840.4 300H183.6c-19.7 0-30.7 20.8-18.5 35l328.4 380.8c9.4 10.9 27.5 10.9 37 0L858.9 335c12.2-14.2 1.2-35-18.5-35z"
                  />
                </svg>
              </a>
              <ul
                class="dropdown-menu"
              >
                <li
                  class="nav media-dropdown-item"
                  role="menuitem"
                >
                  <a
                    href="/blog/1"
                  >
                    <span>
                      Blog
                    </span>
                  </a>
                </li>
                <li
                  class="nav media-dropdown-item"
                  role="menuitem"
                >
                  <a
                    href="/podcast/1"
                  >
                    <span>
                      Podcast
                    </span>
                  </a>
                </li>
              </ul>
            </li>
            <li
              class="dropdown nav "
              role="menuitem"
              style="margin-right: 10px;"
            >
              <a
                aria-expanded="false"
                aria-haspopup="true"
                class="nav"
                data-toggle="dropdown"
                href="#"
                role="button"
              >
                Apply
                <svg
                  class="dropdown-caret"
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  stroke-width="0"
                  viewBox="0 0 1024 1024"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M840.4 300H183.6c-19.7 0-30.7 20.8-18.5 35l328.4 380.8c9.4 10.9 27.5 10.9 37 0L858.9 335c12.2-14.2 1.2-35-18.5-35z"
                  />
                </svg>
              </a>
              <ul
                class="dropdown-menu"
              >
                <li
                  class="nav"
                  role="menuitem"
                >
                  <a
                    href="/apply"
                  >
                    <span>
                      Veteran
                    </span>
                  </a>
                </li>
                <li
                  class="nav"
                  role="menuitem"
                >
                  <a
                    href="/mentor"
                  >
                    <span>
                      Mentor
                    </span>
                  </a>
                </li>
              </ul>
            </li>
            <li
              class="nav"
              role="menuitem"
            >
              <a
                href="/syllabus"
              >
                <span>
                  Syllabus
                </span>
              </a>
            </li>
            <li
              class="nav"
              role="menuitem"
            >
              <a
                href="/jobs"
              >
                <span>
                  Job Search
                </span>
              </a>
            </li>
            <li
              class="nav"
              role="menuitem"
            >
              <a
                href="/contact"
              >
                <span>
                  Contact
                </span>
              </a>
            </li>
            <li
              class="nav"
              role="menuitem"
            >
              <a
                href="/code-of-conduct"
              >
                <span>
                  Code Of Conduct
                </span>
              </a>
            </li>
            <li
              class="donate"
              role="menuitem"
            >
              <a
                href="/donate"
              >
                Donate
              </a>
            </li>
          </ul>
        </div>
      </div>
    </nav>
    <div>
      <section
        class="call-to-action-small"
        id="call-to-action-small"
      >
        <div
          class="container"
        >
          <div
            class="row"
          >
            <div
              class="col-sm-12 text-center"
            >
              <h2>
                Help Us Teach More Veterans How To Code Â 
              </h2>
              <button
                class="donate"
              >
                <a
                  href="/donate"
                >
                  donate
                </a>
              </button>
            </div>
          </div>
        </div>
      </section>
      <section
        class="footer-widgets pad-extra"
      >
        <div
          class="container"
        >
          <div
            class="row"
          >
            <div
              class="col-md-4"
            >
              <div
                class="widgets-content footer-widget-wrap"
              >
                <h3
                  class="widgets-title"
                >
                  About #VetsWhoCode
                </h3>
                <p>
                  Vets Who Code Inc. is an exempt organization as described in Section 501(c)(3) of the Internal Revenue Code. Our EIN is 86-2122804.
                </p>
                <div
                  class="footer-social"
                  style="margin-bottom: 10px;"
                >
                  <a
                    aria-label="Facebook"
                    href="http://bit.ly/vetswhocode-facebook-link"
                  >
                    <i
                      class="fa"
                    >
                      <svg
                        fill="currentColor"
                        height="1em"
                        stroke="currentColor"
                        stroke-width="0"
                        viewBox="0 0 320 512"
                        width="1em"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          d="M279.14 288l14.22-92.66h-88.91v-60.13c0-25.35 12.42-50.06 52.24-50.06h40.42V6.26S260.43 0 225.36 0c-73.22 0-121.08 44.38-121.08 124.72v70.62H22.89V288h81.39v224h100.17V288z"
                        />
                      </svg>
                    </i>
                  </a>
                  <a
                    aria-label="Twitter"
                    href="http://bit.ly/vets-who-code-twitter"
                  >
                    <i
                      class="fa"
                    >
                      <svg
                        fill="currentColor"
                        height="1em"
                        stroke="currentColor"
                        stroke-width="0"
                        viewBox="0 0 512 512"
                        width="1em"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
                        />
                      </svg>
                    </i>
                  </a>
                  <a
                    aria-label="Instagram"
                    href="https://bit.ly/2sJ6W9f"
                  >
                    <i
                      class="fa"
                    >
                      <svg
                        fill="currentColor"
                        height="1em"
                        stroke="currentColor"
                        stroke-width="0"
                        viewBox="0 0 448 512"
                        width="1em"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
                        />
                      </svg>
                    </i>
                  </a>
                  <a
                    aria-label="Linkedin"
                    href="https://bit.ly/2sFKi1u"
                  >
                    <i
                      class="fa"
                    >
                      <svg
                        fill="currentColor"
                        height="1em"
                        stroke="currentColor"
                        stroke-width="0"
                        viewBox="0 0 448 512"
                        width="1em"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          d="M100.28 448H7.4V148.9h92.88zM53.79 108.1C24.09 108.1 0 83.5 0 53.8a53.79 53.79 0 0 1 107.58 0c0 29.7-24.1 54.3-53.79 54.3zM447.9 448h-92.68V302.4c0-34.7-.7-79.2-48.29-79.2-48.29 0-55.69 37.7-55.69 76.7V448h-92.78V148.9h89.08v40.8h1.3c12.4-23.5 42.69-48.3 87.88-48.3 94 0 111.28 61.9 111.28 142.3V448z"
                        />
                      </svg>
                    </i>
                  </a>
                  <a
                    aria-label="Github"
                    href="https://bit.ly/2HSojjv"
                  >
                    <i
                      class="fa"
                    >
                      <svg
                        fill="currentColor"
                        height="1em"
                        stroke="currentColor"
                        stroke-width="0"
                        viewBox="0 0 496 512"
                        width="1em"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                        />
                      </svg>
                    </i>
                  </a>
                </div>
                <a
                  aria-label="Guidestar"
                  href="https://www.guidestar.org/profile/86-2122804"
                >
                  <img
                    alt="guidestar seal"
                    class="footer-guidestar"
                    height="96"
                    src="/images/guidestar.svg"
                    width="96"
                  />
                </a>
              </div>
            </div>
            <div
              class="col-md-4"
            >
              <div
                class="footer-widget-wrap"
              >
                <div
                  class="row"
                >
                  <div
                    class="col-md-11 col-md-offset-1"
                  >
                    <h3
                      class="widgets-title"
                    >
                      Site Map
                    </h3>
                  </div>
                  <div
                    class="col-md-5 col-md-offset-1"
                  >
                    <div
                      class="widgets-content"
                    >
                      <ul
                        class="widgets-list"
                      >
                        <li>
                          <a
                            href="/"
                          >
                            Home
                          </a>
                        </li>
                        <li>
                          <a
                            href="/about"
                          >
                            About
                          </a>
                        </li>
                        <li>
                          <a
                            href="/board"
                          >
                            Board
                          </a>
                        </li>
                        <li>
                          <a
                            href="/syllabus"
                          >
                            Syllabus
                          </a>
                        </li>
                        <li>
                          <a
                            href="/code-of-conduct"
                          >
                            Conduct
                          </a>
                        </li>
                        <li>
                          <a
                            href="/jobs"
                          >
                            Job Search
                          </a>
                        </li>
                        <li>
                          <a
                            href="/testimonials"
                          >
                            Testimonials
                          </a>
                        </li>
                      </ul>
                    </div>
                  </div>
                  <div
                    class="col-md-6"
                  >
                    <div
                      class="widgets-content"
                    >
                      <ul
                        class="widgets-list"
                      >
                        <li>
                          <a
                            href="/blog"
                          >
                            Blog
                          </a>
                        </li>
                        <li>
                          <a
                            href="/apply"
                          >
                            Apply
                          </a>
                        </li>
                        <li>
                          <a
                            href="/donate"
                          >
                            Donate
                          </a>
                        </li>
                        <li>
                          <a
                            href="/mentor"
                          >
                            Mentor
                          </a>
                        </li>
                        <li>
                          <a
                            href="/podcast"
                          >
                            Podcasts
                          </a>
                        </li>
                        <li>
                          <a
                            href="/contact"
                          >
                            Contact Us
                          </a>
                        </li>
                      </ul>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="col-md-4"
            >
              <div
                class="widgets-content"
              >
                <h3
                  class="widgets-title"
                >
                  Contact Us
                </h3>
                <div
                  class="office map"
                >
                  <p>
                    Hi, if you have any questions, please drop us a line.
                  </p>
                  <span>
                    <a
                      aria-label="Email hello@vetswhocode.io"
                      class="footer-span"
                      href="hello@vetswhocode.io"
                      title="Email Contact"
                    >
                      hello@vetswhocode.io
                    </a>
                  </span>
                  <p>
                     Phone: (404) 383-4414
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </section>
      <footer
        class="section footer"
        id="footer"
      >
        <div
          class="container"
        >
          <div
            class="row"
          >
            <div
              class="col-md-6 col-sm-6"
            >
              <p
                class="copyright"
              >
                <svg
                  color="#fff"
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  stroke-width="0"
                  style="color: rgb(255, 255, 255);"
                  viewBox="0 0 512 512"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M256 8C119.033 8 8 119.033 8 256s111.033 248 248 248 248-111.033 248-248S392.967 8 256 8zm0 448c-110.532 0-200-89.451-200-200 0-110.531 89.451-200 200-200 110.532 0 200 89.451 200 200 0 110.532-89.451 200-200 200zm107.351-101.064c-9.614 9.712-45.53 41.396-104.065 41.396-82.43 0-140.484-61.425-140.484-141.567 0-79.152 60.275-139.401 139.762-139.401 55.531 0 88.738 26.62 97.593 34.779a11.965 11.965 0 0 1 1.936 15.322l-18.155 28.113c-3.841 5.95-11.966 7.282-17.499 2.921-8.595-6.776-31.814-22.538-61.708-22.538-48.303 0-77.916 35.33-77.916 80.082 0 41.589 26.888 83.692 78.277 83.692 32.657 0 56.843-19.039 65.726-27.225 5.27-4.857 13.596-4.039 17.82 1.738l19.865 27.17a11.947 11.947 0 0 1-1.152 15.518z"
                  />
                </svg>
                2022
                <span>
                  <a
                    aria-label="Link to Copyright License"
                    class="footer-span"
                    href="https://github.com/Vets-Who-Code/vets-who-code-app/blob/master/LICENSE"
                    rel="noopener noreferrer"
                    target="_blank"
                    title="Copyright"
                  >
                    #VetsWhoCode
                  </a>
                </span>
              </p>
            </div>
            <div
              class="col-md-6 col-sm-6"
            >
              <ul
                class="footer-menu"
              >
                <li>
                  Made with 
                  <svg
                    color="#C5203E"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    style="color: rgb(197, 32, 62);"
                    viewBox="0 0 640 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M192 384h192c53 0 96-43 96-96h32c70.6 0 128-57.4 128-128S582.6 32 512 32H120c-13.3 0-24 10.7-24 24v232c0 53 43 96 96 96zM512 96c35.3 0 64 28.7 64 64s-28.7 64-64 64h-32V96h32zm47.7 384H48.3c-47.6 0-61-64-36-64h583.3c25 0 11.8 64-35.9 64z"
                    />
                  </svg>
                  , 
                  <svg
                    color="#C5203E"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    style="color: rgb(197, 32, 62);"
                    viewBox="0 0 640 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M278.9 511.5l-61-17.7c-6.4-1.8-10-8.5-8.2-14.9L346.2 8.7c1.8-6.4 8.5-10 14.9-8.2l61 17.7c6.4 1.8 10 8.5 8.2 14.9L293.8 503.3c-1.9 6.4-8.5 10.1-14.9 8.2zm-114-112.2l43.5-46.4c4.6-4.9 4.3-12.7-.8-17.2L117 256l90.6-79.7c5.1-4.5 5.5-12.3.8-17.2l-43.5-46.4c-4.5-4.8-12.1-5.1-17-.5L3.8 247.2c-5.1 4.7-5.1 12.8 0 17.5l144.1 135.1c4.9 4.6 12.5 4.4 17-.5zm327.2.6l144.1-135.1c5.1-4.7 5.1-12.8 0-17.5L492.1 112.1c-4.8-4.5-12.4-4.3-17 .5L431.6 159c-4.6 4.9-4.3 12.7.8 17.2L523 256l-90.6 79.7c-5.1 4.5-5.5 12.3-.8 17.2l43.5 46.4c4.5 4.9 12.1 5.1 17 .6z"
                    />
                  </svg>
                  , and
                   
                  <svg
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 512 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M32 0C14.3 0 0 14.3 0 32v464c0 8.8 7.2 16 16 16h32c8.8 0 16-7.2 16-16V32C64 14.3 49.7 0 32 0zm267.9 303.6c-57.2-15.1-111.7-28.8-203.9 11.1V384c185.7-92.2 221.7 53.3 397.5-23.1 11.4-5 18.5-16.5 18.5-28.8v-36c-43.6 17.3-80.2 24.1-112.1 24.1-37.4-.1-68.9-8.4-100-16.6zm0-96c-57.2-15.1-111.7-28.8-203.9 11.1v61.5c94.8-37.6 154.6-22.7 212.1-7.6 57.2 15.1 111.7 28.8 203.9-11.1V200c-43.6 17.3-80.2 24.1-112.1 24.1-37.4 0-68.9-8.3-100-16.5zm9.5-125.9c51.8 15.6 97.4 29 202.6-20.1V30.8c0-25.1-26.8-38.1-49.4-26.6C291.3 91.5 305.4-62.2 96 32.4v151.9c94.8-37.5 154.6-22.7 212.1-7.6 57.2 15 111.7 28.7 203.9-11.1V96.7c-53.6 23.5-93.3 31.4-126.1 31.4s-59-7.8-85.7-15.9c-4-1.2-8.1-2.4-12.1-3.5V75.5c7.2 2 14.3 4.1 21.3 6.2zM160 128.1c-8.8 0-16-7.1-16-16 0-8.8 7.2-16 16-16s16 7.1 16 16-7.2 16-16 16zm0-55.8c-8.8 0-16-7.1-16-16 0-8.8 7.2-16 16-16s16 7.1 16 16c0 8.8-7.2 16-16 16zm64 47.9c-8.8 0-16-7.1-16-16 0-8.8 7.2-16 16-16s16 7.1 16 16c0 8.8-7.2 16-16 16zm0-55.9c-8.8 0-16-7.1-16-16 0-8.8 7.2-16 16-16s16 7.1 16 16c0 8.8-7.2 16-16 16z"
                    />
                  </svg>
                   by the veterans of #VetsWhoCode.
                </li>
              </ul>
            </div>
          </div>
        </div>
      </footer>
    </div>
  </main>
</div>
`;
